docs/sdks/swift-sdk/integrate-with-auth0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Integrate With Auth0
+sidebar_position: 3
+---
+
+This guide describes how to configure Auth0 to delegate to Beyond Identity for authentication during an OAuth2 authorization flow. 
+
+### Prerequisites
+
+ - [Integrate With Auth0](/guides/sso-integrations/integrate-with-auth0)
+ - [Swift SDK Overview](overview)
+
```

Comment:
Can you add the images that @Will5 created to the Auth0 and Okta guides?

Reply:
Updated and added an SDK section like android